profiles/postgres_log/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "version": "1",
+  "engines" : {
+    "flink" : {
+      "connectors": {
+        "postgres_log-source": {
+          "connector": "postgres-cdc",
+          "hostname": "postgres_log",
+          "port": "5432",
+          "username": "${JDBC_USERNAME}",
+          "password": "${JDBC_PASSWORD}",
+          "database-name": "datasqrl",
+          "schema-name": "public",
+          "table-name": "${sqrl:table}",
+          "slot.name": "flink_slot",
+          "decoding.plugin.name": "pgoutput",
+          "debezium.slot.drop_on_stop": "false"
+        },
+        "postgres_log-sink": {
+          "connector": "jdbc-sqrl",
+          "password": "${JDBC_PASSWORD}",
+          "driver": "org.postgresql.Driver",
+          "username": "${JDBC_USERNAME}",
+          "url": "${POSTGRES_LOG_JDBC_URL}",
```

Comment:
Did you try merging the engines so there only needs to be one postgres instance? I'm curious what issues you hit.

Reply:
I didn't hit any issues, it works with one postgres instance as well. It depends only on the profile, but I felt like this is a cleaner design.